### ISSUE TITLE
Vax for the Win - Six Flags 404

### ISSUE THREAD

**Issue by m-sullivan7** [cagov/covid19#6173]
It looks like the _Vax for the Win - Six Flags_ page has been archived. That URL currently provides a 404 error. We may want to update this:

- Redirect to the main _Vax for the Win_ page, like we do for the _Dream Vacations_ URL.
- Send people to the 410 page so search engines know there will no longer be any content at this page. This is best practice if we do not have an appropriate redirect.

This issue is not urgent.

**Comment by mediajunkie** [comment]
yes, let's please redirect to the main page.

**Comment by m-sullivan7** [comment]
I went to create the redirects. We already have them for this page (as I should have known given that Peggy was in charge of the page).

I reviewed my browser history and looked at the [Google Doc](https://docs.google.com/document/d/1TTmCBFSki16o0mChpcvXLHSDFSn9XcNtDqjqeBY8dBM/edit?usp=sharing) where I found the link that inspired this issue. It turns out that link went to the staging site. I assume redirects do not work on the staging site, nor do they need to. I've confirmed the production link properly redirects. Therefore, I'm closing this issue as no work is needed.
